config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -59,10 +59,41 @@ periodics:
     testgrid-alert-email: kubernetes-release-team@googlegroups.com
     description: 'Ends up running: make quick-release'
 - interval: 4h
-  name: ci-kubernetes-build-debian-unstable
+  name: ci-release-build-packages-debs
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
+  annotations:
+    testgrid-alert-email: kubernetes-release-managers@googlegroups.com
+    testgrid-dashboards: sig-release-master-informing
+    testgrid-tab-name: build-packages-debs
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20190703-1f4d616
+      args:
+      - --repo=k8s.io/release
+      - --root=/go/src
+      - --timeout=180
+      - --scenario=execute
```

Comment:
The use of bootstrap is super deprecated, what is it about this image you require?

Check out the `periodic` example here for what to use instead (the `extra_refs` thing)

https://github.com/kubernetes/test-infra/tree/master/config/jobs#job-examples